x-pack/legacy/plugins/ml/server/models/calendar/calendar_manager.ts

```diff
@@ -6,17 +6,46 @@
 
 import _ from 'lodash';
```

Comment:
it looks like only `difference` is used from lodash

Reply:
Updated to only pull in `difference` e4c0e64639120c16c78f85258334091d4ec0c6ea